src/dropdown/dropdown.js

```diff
@@ -170,9 +162,50 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
         return;
       }
       dropdownCtrl.dropdownMenu = element;
+
+      var dropdownMenu = element,
+          dropdown = element.parent(),
```

Comment:
That's true to the extent if Bootstrap usage is assumed - I know there are some UI Bootstrap users out there who do not use Bootstrap and use the library, even though the library does not officially support that situation.

It still is worth noting.